src/main/java/net/apnic/rdap/authority/config/AuthorityConfiguration.java

```diff
@@ -221,20 +232,15 @@ private void setupAuthorityStore()
 
             if(aConfig.getServers() != null)
             {
-                List<URI> servers = aConfig.getServers().stream()
-                    .map((String strURI) ->
-                    {
-                        try
-                        {
-                            return new URI(strURI);
-                        }
-                        catch(URISyntaxException ex)
-                        {
-                            throw new RuntimeException(ex);
-                        }
-                    })
-                    .collect(Collectors.toList());
-                authority.addServers(servers);
+                authority.addServers(
+                    aConfig.getServers().stream()
+                    .map(URI::create)
+                    .collect(Collectors.toList()));
+            }
+
+            if(aConfig.getDefaultServer() != null)
+            {
+                authority.setDefaultServer(URI.create(aConfig.getDefaultServer()));
             }
 
             authorityStore().addAuthority(authority);
```

Comment:
In this instance I don't think it would work to well.

As an example this configuration code is translating from one schema to another. In yaml its valid to add a null value for a stub but I don't expect that to be passed into a valid object. You could add the idea of a isValid() method to the POJO but for this case I don't think the idea is great because 1) Its always a point of code that will require updating every time something is added to the authority that contributes to state, 2) In this instance defaultServer can be null in the config and doing so would get the whole object invalidated where as the config code can just throw the value out and still maintain a valid Authority.

Hope that make sense.